src/OrchardCore/OrchardCore.Abstractions/Json/JOptions.cs

```diff
@@ -31,10 +31,11 @@ public static class JOptions
 
     static JOptions()
     {
+        Base.Converters.Add(new LoginInfoJsonConverter());
```

Comment:
work fine

Reply:
but still exists other problems~~

Reply:
What works fine and What other problems?